console/v1/types.go

```diff
@@ -7,3 +7,19 @@ type Link struct {
 	// href is the absolute secure URL for the link (must use https)
 	Href string `json:"href"`
 }
+
+//  Represents ability to provide custom pod log links.
+type LogLink struct {
+	// text is the display text for the link
+	Text string `json:"text"`
+	// Absolute secure URL (must use https) for the log link including variables
+	// to be replaced. Supported variables, which must exactly match, include:
```

Comment:
We need to be more explicit about the exact format of the variables in the template. The example hints at it, but better to make it very clear.